releases/UWP/README.md

```diff
@@ -20,7 +20,7 @@ When using Visual Studio this package requires Visual Studio 2017 or later.
 
 ### 5.3.1 (March 7, 2017)
 
-- Fixes [DataContractJsonSerializer failing in Microsoft.NetCore.UniversalWindowsPlatform 5.3.0](https://github.com/dotnet/corefx/issues/10374)
+- Fix a [bug](https://github.com/dotnet/corefx/issues/10374) that caused DataContractJsonSerializer fail to serialize any [DataContract] type whose default constructor is not public.
```

Comment:
> Fix a [bug](https://github.com/dotnet/corefx/issues/10374) that caused DataContractJsonSerializer fail to serialize 

I would change this to

Fixes a [bug](https://github.com/dotnet/corefx/issues/10374) that caused DataContractJsonSerializer to fail to serialize